server/config/persist_options.go

```diff
@@ -35,17 +36,30 @@ import (
 	"go.etcd.io/etcd/clientv3"
 )
 
+// instanceOptions are all instance-level configurations that need to persist to storage and
+// allows to access them safely.
+type instanceOptions struct {
+	enableLocalTSO bool
+	labels         map[string]string
+}
+
 // PersistOptions wraps all configurations that need to persist to storage and
 // allows to access them safely.
 type PersistOptions struct {
+	clusterVersion unsafe.Pointer
 	// configuration -> ttl value
-	ttl             *cache.TTLString
+	ttl *cache.TTLString
+	// cluster-level configuration
 	schedule        atomic.Value
 	replication     atomic.Value
 	pdServerConfig  atomic.Value
 	replicationMode atomic.Value
 	labelProperty   atomic.Value
-	clusterVersion  unsafe.Pointer
+	// instance-level configuration
+	instance struct {
```

Comment:
The main reason for persist instance config is the dynamic setting？But I think users rarely change this configration, and restarting the instance can warn users of important changes.

Reply:
Yes, the main reason to introduce the persisted instance-level config is to provide dynamic changing APIs, which the TiPocket test case needs also. According to this, I think making it persist is more reasonable for both test and useful purposes.

Reply:
The other point is if persist, to change the config should use API or PD control. It's strange for users about the instance level configuration because it's not works in instance configuration. I think should modify the test instead of causing some user behavior problems.

Reply:
cc @rleungx @disksing

Reply:
If this is only used for the tests, it is unnecessary to introduce the instance level config which may confuse the user when using the API.